### PR TITLE
Bugfix/INTG-627 Signature Flow PosRefID

### PR DESCRIPTION
**Pull request by GeorgeMcLellan**
Add in posRefId as a parameter for signature approve and decline.

https://mx51.atlassian.net/browse/INTG-627

### CODE REVIEW

File: Library/SPIClient.m [hunk 1]
```diff
@@ -535,7 +535,8 @@ - (void)initiateRefundTx:(NSString *)posRefId
     });
 }
 
-- (void)acceptSignature:(BOOL)accepted {
+- (void)acceptSignature:(NSString *)posRefId
```
Comment (rossi-mx51) [inline Library/SPIClient.m:538]: the posRefId should already exist in `weakSelf.state.txFlowState` object. so I think  we can simply access it by `weakSelf.state.txFlowState.posRefId`

Comment (rossi-mx51) [inline Library/SPIClient.m]: the posRefId should already exist in weakSelf.state.txFlowState object. so I think we can simply access it by weakSelf.state.txFlowState.posRefId

**Review by rossi-mx51 — CHANGES_REQUESTED** [review]

File: Library/SPIClient.m (hunk 1 above)
Comment (GeorgeMcLellan) [reply Library/SPIClient.m]: Cheers @rossi-mx51 how does it look now? Also, is there any way for me to test the changes I've made?

Comment (rossi-mx51) [reply Library/SPIClient.m]: Looks better now, thanks @GeorgeMcLellan.  In terms of testing it. We're having a repo for sample POSes that use SPI iOS  [here](https://github.com/mx51/spi-samples-ios).   I'm not sure how to run those TBH, but I'm sure you can figure it out :)

**Review by rossi-mx51 — APPROVED** [review]

File: Library/SPIClient.m (hunk 1 above)
Comment (GeorgeMcLellan) [reply Library/SPIClient.m]: Yep, i've got them cloned, just wondering if we need to test this before merging and passing over to QA? Just not sure on the whole process over here. I've increased the version number to 2.9.2, I believe David from QA will test it?

**Pull request merged**